File: Java/Advanced Conditional Statements/Exercise/ACS Exercise/src/Journey.java
import java.util.Scanner;

public class Journey {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double budget = Double.parseDouble(input.nextLine()), expenses = 0;
        String season = input.nextLine(), dest = "", restType = "";

        if(budget <= 100){
            //somewhere in Bulgaria
            dest = "Bulgaria";
            switch(season){
                case "summer":
                    restType = "Camp";
                    expenses = budget * 0.3;
                    break;
                case "winter":
                    restType = "Hotel";
                    expenses = budget * 0.7;
                    break;
            }
        }else if(budget <= 1000){
            dest = "Balkans";
            switch(season){
                case "summer":
                    restType = "Camp";
                    expenses = budget * 0.4;
                    break;
                case "winter":
                    restType = "Hotel";
                    expenses = budget * 0.8;
                    break;
            }
        }else {
            dest = "Europe";
            expenses = budget * 0.9;
            restType = "Hotel";
        }

        System.out.printf("Somewhere in %s\n%s - %.2f", dest, restType, expenses);
    }
}

File: Python/Basics/Conditional Statements Advanced/Exercise/Journey.py
budget = float(input())
season = input() #summer / winter

destination = ""
place = ""
spentMoney = 0

if budget <= 100:
    destination = "Bulgaria"
    if season == "winter":
        spentMoney = budget * 0.7
        place = "Hotel"
    elif season == "summer":
        spentMoney = budget * 0.3
        place = "Camp"

elif 100 < budget <= 1000:
    destination = "Balkans"
    if season == "winter":
        spentMoney = budget * 0.8
        place = "Hotel"
    elif season == "summer":
        spentMoney = budget * 0.4
        place = "Camp"
else:
    destination = "Europe"
    spentMoney = budget * 0.9
    place = "Hotel"

print(f"Somewhere in {destination}\n{place} - {spentMoney:.2f}")
